src/main/java/com/sigma/cinematicketpro/dto/AuthenticationRequest.java

```diff
@@ -0,0 +1,16 @@
+package com.sigma.cinematicketpro.dto;
+
+import jakarta.validation.constraints.NotNull;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class AuthenticationRequest {
+    @NotNull
+    private String username;
+    @NotNull
```

Comment:
added check for username size and @NotBlank for password. The decision to leave the password validation less strict was made to avoid possible security vulnerabilities.